### PR TITLE
Added SRM Framework Configuration BPA documentation

### PR DESCRIPTION
**Pull request by JorgeFilipeDias**
Documentation for BPA SRM Framework Configuration, can you please take a look

### CODE REVIEW

File: user-guide/Advanced_Functionality/DataMiner_Systems/BPA_tests/BPA_SRM_Framework_Configuration.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+## Results
+
+### Success
+
+* "No invalid Bookings detected in the system.*
+
+No inconsistencies have been detected in the system.
+
+### Error
```
Comment (MariekeGO) [inline user-guide/Advanced_Functionality/DataMiner_Systems/BPA_tests/BPA_SRM_Framework_Configuration.md:26]: @JorgeFilipeDias, I notice that these error messages are different from the ones you sent me for review via email earlier, which seems a bit odd. Are these additional messages that still need to be reviewed?

Comment (JorgeFilipeDias) [reply user-guide/Advanced_Functionality/DataMiner_Systems/BPA_tests/BPA_SRM_Framework_Configuration.md:26]: The ones in the mail were the actual code, the messages that the user will see.
Here we have a complete list of the properties with the explanation of the error.
E.g.: For '"Booking doesn't have Property with names: 'Booking Manager', 'Booking Life Cycle', 'FriendlyReference', 'PreRoll', 'PostRoll', 'Service State', 'Start', 'End', 'VisualBackground' and 'VisualForeground'"' what the user will see is ': Booking ID: 8cf046c2-a41b-4bbc-bbe2-7c53eff8fe03 Name: Example Booking does not contain ‘Start’ property'

File: user-guide/Advanced_Functionality/DataMiner_Systems/BPA_tests/BPA_SRM_Framework_Configuration.md [hunk 2]
```diff
@@ -4,52 +4,59 @@ uid: BPA_SRM_Framework_Configuration
...
+If the test fails to execute, this message is displayed:
+
+- `Resource Manager is not initialized.`
 
 ## Impact when issues detected
 
-- Impact: "Bookings may not execute correctly."
-- Action: "Please contact 'support.automation-orchestration@skyline.be' for support."
+- Impact: Bookings may not execute correctly.
+- Action: Please contact <support.automation-orchestration@skyline.be> for support.
```
Comment (JorgeFilipeDias) [inline user-guide/Advanced_Functionality/DataMiner_Systems/BPA_tests/BPA_SRM_Framework_Configuration.md:56]: Should be 'Please contact your system administrator for support.', forgot to change that here, my mistake

**Review by JorgeFilipeDias — COMMENTED** [review]
Just one thing, that I forgot earlier.
The rest looks good to me

**Review by MariekeGO — APPROVED** [review]

**Pull request merged**